wealth-management-client/src/app/data/stocks-recommendations.json

```diff
@@ -39,15 +39,6 @@
     "rcomBy": "Hemanth",
```

Comment:
Please change AVANTI to AVANTIFEED.
SHARDA to SHARDAMOTR
PREMCO is not listed in NSE

Reply:
OK.
Do we need a different query for BSE stock only? If yes, lets add it to backlog.

Reply:
Done.
Have removed PREMCO from now.